tests/test_replication.py

```diff
@@ -163,18 +163,20 @@ def test_start_and_recover_from_error(self):
         cur = conn.cursor()
 
         self.create_replication_slot(cur, output_plugin='test_decoding')
-
-        # try with invalid options
-        cur.start_replication(
-            slot_name=self.slot, options={'invalid_param': 'value'})
+        self.make_replication_events()
 
         def consume(msg):
-            pass
-        # we don't see the error from the server before we try to read the data
```

Comment:
This was the behavior with older server versions, which is corrected in newer ones, e.g. v11.  No reason to test specific server behavior if we care about the client being able to recover from error.